backend/inc/fnc.con.php

```diff
@@ -730,15 +730,15 @@ function con_create_site_meta_from_idcatside($idclient, $idlang_current, $idcats
 	//create idside
 	$sql = "INSERT INTO ".$cms_db['side']." (idclient) VALUES ('$idclient')";
 	$db->query($sql);
-	$idside_to = mysqli_insert_id($this->db->Link_ID);
+	$idside_to = mysqli_insert_id($GLOBALS['db']->Link_ID);
```

Comment:
Why not using `$db->Link_ID`?

Reply:
$this->db is not set in this function. $GLOBALS['db'] is reliable.